src/app/static/src/app/store/root/getters.ts

```diff
@@ -23,9 +23,11 @@ export const getters: RootGetters & GetterTree<RootState, RootState> = {
 
     warnings: (state: RootState) => (stepName: string): StepWarnings => {
         const filterWarnings = (warnings: Warning[], stepLocation: string) =>
-            (warnings || []).filter(warning => warning.locations.some(location => location === stepLocation))
+            stepLocation ?
+                (warnings || []).filter(warning => warning.locations.some(location => location === stepLocation)) :
```

Comment:
I am still unsure why `(warnings || [])` is needed 

Also this `stepLocation ?`